cmd/criticize/main.go

```diff
@@ -144,6 +152,33 @@ func parseArgv(l *linter) {
 	}
 }
 
+func (l *linter) LoadConfig() {
+	if _, err := os.Stat(l.configFile); err != nil {
+		return
+	}
+
+	raw, err := ioutil.ReadFile(l.configFile)
+	if err != nil {
+		log.Fatalf("cannot read config file %s, got error: %s", l.configFile, err)
+		return
```

Comment:
Hmm.. Seems like a check candidate.
Return after Fatal-like logging (same for panic logging?).

Reply:
Hm...it should be warning here ;D

Reply:
Yep.